tests/run-arm-smoke-tests.sh

```diff
@@ -27,18 +28,52 @@ function run_tests {
     do
         test_case="scalers/${scaler}"
         counter=$((counter+1))
-        ./node_modules/.bin/ava $test_case > "${test_case}.log" 2>&1 &
+        ./node_modules/.bin/ava $test_case > "${test_case}.1.log" 2>&1 &
```

Comment:
Do we need to do this? Isn't enough just to rename the retry test logs?

Reply:
It's the same approach as normal e2e test, but I can change in both sides.
Do you prefer something like 'retry-xxxxxxx.test.ts'?

Reply:
Yeah 👍